source/index.html.erb

```diff
@@ -1 +1,31 @@
-<h1 class="heading-large">GOV.UK Design System</h1>
+<main id="content" role="main" class="govuk-u-font-smoothing">
```

Comment:
If we put this on the `main` tag we'll have to add it to every different page that we create, which means we might end up with a lot of tidying up to do later - could it go somewhere else? For example, the body?

Reply:
I think `class="govuk-u-font-smoothing"` should be on `body` as discussed

Reply:
👍  Done!